app/src/main/java/com/rahobbs/todo/activities/SingleFragmentActivity.java

```diff
@@ -0,0 +1,33 @@
+package com.rahobbs.todo.activities;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v7.app.AppCompatActivity;
+
+import com.rahobbs.todo.R;
+
+import static android.support.v4.app.NavUtils.navigateUpFromSameTask;
+
+/**
+ * Created by rachael on 2/2/16.
```

Comment:
Total nitpick comment here, I know Android Studio puts these all in everywhere, but I would delete them (:scissors:) them. I know this is just your app, but when you work on a team it's a good habit to just delete these